### ISSUE TITLE
[MHV SM / Accessibility] Message Filter heading and button do not respond to text size settings

### ISSUE THREAD

**Issue by steveg-IAT** [department-of-veterans-affairs/va.gov-team#128468]
### 📝 User story

As a user,
I have adjusted my browser default text to be 24px 
so that I can easily read the text on the page.

### 📝 Problem statement
Similar to #122721 

When using the browser's default font size settings and adjusting that to 24px base, several items within the message filter component do not scale up with the base font setting...

- **The filter heading**: 
  - font-size is hardcoded to 20px
  - line-height is hardcoded to 26px
- **The 'Show filters' accordion button**:
  - Button width is hardcoded to 166px, which forces the text to wrap poorly
  - "+" button decorator stays in it's fixed position, overlaps the text

- **SIDE ISSUE for discussion**: (related to #125156) This component does not match the functions of the similar component, "What's a message ID?", that is right above it.
   - The accordion button/triggers are full-width of the containing element, and I can activate each by clicking anywhere to the right of the elements...  
    - "What is a message ID" component:
      - Has a focus indicator that spans the whole click area
      - Hover state triggers and spans the whole click area
      - Has a time-based transition open
    - "Show filtes" component:
       - The focus area only indicates the button and only displays when the actual button is clicked
       - Hover state is only triggered and spans when over the button directly
       - Because this element is a button, the alignment is visually off due to the extra margin and padding that is applied.
       - Does not have a time-based transition open

### 🌌 Supporting screenshots or videos

<img width="804" height="713" alt="Image" src="https://github.com/user-attachments/assets/90d42969-4d37-4bf8-9e13-cf45f1a8f355" />

### 👣 Reproduction steps

1. As [Test User #4](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/teams/sm/test-accounts.md)
2. Set the default font-size (base) to (large) 24px or greater;
3. > Notice that several elements within the filter card components do not respond to the default size appropriately.

### 🔗 Point of contact

@steveg-IAT

### ⚠️ Severity level

4, Minor. Consider fixing or exploring in 2-4 sprints post-launch.

### 🔎 Relevant standards

- [SC 1.4.4 Resize Text](https://www.w3.org/TR/WCAG22/#resize-text)

### 𝌣 Proposed solution or next steps

- Change font sizes to use rem
- Change button width to use rem or %

### ✅ Acceptance criteria

- The fonts should scale relative to the base settings provided by user's browser or system settings.

### ✅ Possible tasks

- [ ] Review, refine, and acknowledge feedback.
- [ ] Update ticket information (title, assignee, labels, project(s), etc.)
- [ ] Fix and/or document decisions made.
- [ ] Product manager to move ticket in their respective product board to 'QA/A11y Review' after reviewing documented decisions / validating fix.
- [ ] Accessibility specialist will close the ticket after reviewing documented decisions / validating the fix.

**Comment by steveg-IAT** [comment]
@6StringCode I'll break the accordion-specific bits into a new ticket for further discussion along with the parent ticket @sarahhortonadhoc has linked.
